src/main/resources/templates/login.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Login</title>
+</head>
+<body>
+    <h1>Let's get this over with and Login!</h1>
```

Comment:
Somewhat redundant.  You could make one template that is used for both login and create.